app/models/ems_refresh/save_inventory_helper.rb

```diff
@@ -33,6 +33,66 @@ def fetch(hash)
     end
   end
 
+  def store_ids_for_new_dto_records(records, dto_collection)
```

Comment:
@Ladas how easy would it be to move the added stuff in here to the `ManagerRefresh` Namespace?